exporter/mezmoexporter/utils.go

```diff
@@ -0,0 +1,37 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mezmoexporter
+
+import "math/rand"
+
+// truncateString Truncates the given string to a maximum length provided by max.
+func truncateString(s string, max int) string {
+	if len(s) < max {
+		return s
+	}
+
+	return s[:max]
+}
+
+const letters = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+// randString Returns a random string of the specified length.
+func randString(n int) string {
```

Comment:
can we move this to a `_test.go` file so its clear it is used for testing only?

Reply:
Good catch.  Will move to `utils_test.go`.